<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>619</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="rowCount">
        <number>0</number>
       </property>
       <column>
        <property name="text">
         <string>InstrumentID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>TradingDay</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>UpdateTime</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>UpdateMillisec</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>LastPrice</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Volume</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>OpenInterest</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>BidPrice1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>BidVolume1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>AskPrice1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>AskVolume1</string>
        </property>
       </column>
      </widget>
      <widget class="QListWidget" name="listWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>system</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionConfig">
   <property name="text">
    <string>config...</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>start...</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>api version</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
